sporkify/frontend/templates/inventory.html

```diff
@@ -15,11 +15,17 @@ <h1>Warehouse Inventory Management</h1>
           <div id="add_item" style="display:none;">
               <h2>Add Item</h2><br>
               <input type="text" name="Item Name" value="Item Name"><br>
-              <h2>Sales Channel</h2><br>
+              <h2>Sales Site</h2><br>
               <form>
-                <input type="radio" name="Sales Channel" value="ebay" checked>Ebay<br>
-                <input type="radio" name="Sales Channel" value="amazon">Amazon<br>
-                <input type="radio" name="Sales Channel" value="self">Self<br>
+                {% for channel in channels %}
+                <input type="radio" name="Sale Site" value="{{channel.name}}">{{channel.name}}<br>
```

Comment:
Here, and on line 27, I'd use `channel.pk` for the `value` field. This will ensure that the unique primary key is provided to the backend when the user submits the form. In these cases, the name's are unique, but it'll be easier to use `pk` on the backend when we implement the logic to actually add an item.